packages/social-previews/package.json

```diff
@@ -45,6 +45,7 @@
 		"enzyme": "^3.11.0"
 	},
 	"peerDependencies": {
-		"react": "^16.12.0"
+		"react": "^16.12.0",
+		"node-sass": ">= 4.x"
```

Comment:
Requiring `node-sass` may be too specific (there are other sass compilers, like `dart-sass` which is the webpack recommendation) and not enough (users still need to configure their webpack installation to compile sass files).

What do you think about not adding any dependency, and instead adding a README.md explaining they need to configure [sass-loader](https://webpack.js.org/loaders/sass-loader/)?

Reply:
Sounds like a good idea to not be restrictive and to let users use any sass package they'd like.  I'll remove the peerDependency and add a note in the README.

When I tested with a new `create-react-app`, simply adding the node-sass package was enough without changing any config - but I suspect `create-react-app` may have been adding a loader for me behind the scenes to make things easier.